src/evict/evict_inline.h

```diff
@@ -332,11 +288,12 @@ __wti_cache_hs_dirty(WT_SESSION_IMPL *session)
 }
 
 /*
- * __wt_cache_eviction_check --
+ * __wt_evict_app_assist_worker_check --
```

Comment:
"app assist" is new terminology. Does it refer to when we exceed triggers and pull app threads into eviction?

Reply:
That's correct, I am pretty sure this function is not called by an internal thread, only app thread.